client/my-sites/stats/hooks/use-notice-visibility-query.ts

```diff
@@ -14,7 +15,7 @@ const DEFAULT_SERVER_NOTICES_VISIBILITY = {
 	// TODO: Check if the site needs to be upgraded to a higher tier on the back end.
 	tier_upgrade: true,
 	gdpr_cookie_consent: false,
-	able_to_submit_user_feedback: true,
+	[ NOTICE_KEY_FOR_FEEDBACK_SUBMISSION ]: true,
 	[ NOTICES_KEY_SHOW_FLOATING_USER_FEEDBACK_PANEL ]: true,
```

Comment:
We should probably keep the const naming consistent. I used `NOTICES_KEY` and you've used `NOTICE_KEY`. I'd also like the keys to be a bit more descriptive. If possible, maybe `NOTICES_KEY_ABLE_TO_SUBMIT_FEEDBACK`?